src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20210820111534_EES2581AddEmailSentColumnToUserReleaseInviteTable.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES2581AddEmailSentColumnToUserReleaseInviteTable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<bool>(
```

Comment:
As per my other comment, this might actually be a bit more useful as a nullable DateTime.

Reply:
I'll resolve this comment and leave the other comment open and resolve that when this is done.